packages/schematics/update/update/index.ts

```diff
@@ -865,10 +865,7 @@ export default function(options: UpdateSchema): Rule {
       try {
         return isPkgFromRegistry(name, specifier);
       } catch {
-        // Abort on failure because package.json is malformed.
-        throw new SchematicsException(
-          `Failed to parse dependency "${name}" with specifier "${specifier}"`
-              + ` from package.json. Is the specifier malformed?`);
+        return false;
```

Comment:
Could we add a warning here so that the user knows that the dependency will not be used during the version analysis?

Reply:
Added.